xc/xenops_server_xen.ml

```diff
@@ -2354,8 +2354,9 @@ module Actions = struct
 		device_watches := IntMap.add domid [] !device_watches
 
 	let domain_disappeared xc xs domid =
+		let domain = Printf.sprintf "domain/%d" domid in
```

Comment:
How about we pull out the sprintf into a function as well instead of defining it in three places - something like `token_of_domain` ?